page-templates/libertyjs-2017-homepage.php

```diff
@@ -91,6 +91,10 @@ class="ljs2017-header__info-logo"
 		<div class="ljs2017-keynotes__heading">
 			<h2>KEYNOTE</h2>
 		</div>
+		<div class="ljs2017-grid__heading-all">
+			<a href="/speakers">VIEW FULL SPEAKER LINE UP</a>
```

Comment:
If we are going to gradually roll out speakers should we remove the word "full"?